BlueHarvest/ProfileFormatter.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ProfileFormatter.swift
+//  BlueHarvest
+//
+//  Created by David Nutter on 3/19/17.
+//  Copyright © 2017 David Nutter. All rights reserved.
+//
+
+import Foundation
+
+struct ProfileFormatter {
+    
+    struct ProfileKeys {
```

Comment:
Yes that's true. I was mulling over whether to include an initializer via JSON for `Profile` instead of separating this out into the `ProfileFormatter`.